src/redux/configureStore.js

```diff
@@ -10,7 +9,7 @@ const reducer = combineReducers({
 
 const store = createStore(
   reducer,
-  applyMiddleware(logger),
+  applyMiddleware(logger, thunk),
```

Comment:
- [ ] Nice one. However, currently, No data from the API requests is been stored in redux store. I should be able to see the `booksReducer` actions even before making changes to add or remove books. See screen-shot of the current state of the redux store:

![image](https://user-images.githubusercontent.com/51296741/147275069-2fc6c93d-5e49-48d4-85df-e9df57a4acb4.png)

It will be nice for you to fix this.

Reply:
Hi Sir, I took a look at your recommendation, which is a good one. But I have some doubts about it. First, in my project, I can not find some of your screen shoots actions. Second, I tried to replicate loading the page before adding or removing books and I saw that the booksReducer is working and storing the items.
![Screen Shot 2021-12-23 at 13 44 53](https://user-images.githubusercontent.com/61840552/147281042-8da9f46e-d16a-43ab-9bf3-cd9f96025c49.png)

Thanks Sir.

Reply:
Hi JasemDuncan,
I'm sure it is fixed now. The original screenshot was what was been displayed in my browser.